app/controllers/chatController.js

```diff
@@ -30,4 +30,11 @@ exports.createChat = (req, res) => {
             res.send("Chat is already exist");
         }
     });
-};
\ No newline at end of file
+};
+exports.removeChat = (req, res) => {
+    Chat.findById(req.params.id).then(data => {
```

Comment:
@adilet-reactjs I think it will not work as expected
Please refer to documentation and use https://mongoosejs.com/docs/api.html#query_Query-findOneAndRemove this one